integration_tests/two_sided_pay_as_clear_market.feature

```diff
@@ -18,35 +20,42 @@ Feature: Two sided pay_as_clear market tests
   Scenario: All Offers & Bids are traded with finite energy
      Given we have a scenario named two_sided_pay_as_clear/default_2a
      And d3a is installed
+     And d3a uses an two-sided-pay-as-clear market
      When we run the simulation with setup file two_sided_pay_as_clear.default_2a and parameters [24, 60, 60, 1]
      Then all traded energy have finite value
 
   Scenario: Test offers, bids and market-clearing-rate are exported
      Given we have a scenario named two_sided_pay_as_clear/default_2a
      And d3a is installed
+     And d3a uses an two-sided-pay-as-clear market
      When we run the d3a simulation on console with two_sided_pay_as_clear.default_2a for 2 hrs
      Then there are files with offers, bids & market_clearing_rate for every area
 
+  @slow
   Scenario: Supply Demand Curve
      Given we have a scenario named two_sided_pay_as_clear/test_clearing_energy
      And d3a is installed
+     And d3a uses an two-sided-pay-as-clear market
      When we run the d3a simulation on console with two_sided_pay_as_clear.test_clearing_energy for 24 hrs
      Then the export functionality of supply/demand curve is tested
 
   Scenario: One Offer and One Bid are matched at the Bid Rate
      Given we have a scenario named two_sided_pay_as_clear/one_offer_one_bid
      And d3a is installed
+     And d3a uses an two-sided-pay-as-clear market
      When we run the simulation with setup file two_sided_pay_as_clear.one_offer_one_bid and parameters [24, 60, 60, 1]
      Then one-on-one matching of offer & bid in PAC happens at bid rate
 
   Scenario: One Offer and multiple Bids are matched at the last clearing bid rate
      Given we have a scenario named two_sided_pay_as_clear/one_offer_multiple_bids
      And d3a is installed
+     And d3a uses an two-sided-pay-as-clear market
```

Comment:
Why you need this extra step and do not change it in the setup file, like we did before? These setup files are already put into separate folders that indicate their market type. I suggest either to remove these folders and only have one setup file for all market types and change the market type with this extra step OR to change the market type in the setup file. Thanks!

Reply:
Yes, just retested, changing the market type in setup file is enough.